indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/SequentialParallelIndexStatsReporter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.common.task.batch.parallel;
+
+import org.apache.druid.indexing.common.task.IndexTask;
+
+import java.util.Collections;
+
+public class SequentialParallelIndexStatsReporter extends ParallelIndexStatsReporter
+{
+  @Override
+  ParallelIndexStats report(
+      ParallelIndexSupervisorTask task,
+      Object runner,
+      boolean includeUnparseable,
+      boolean full
+  )
+  {
+    IndexTask currentSequentialTask = (IndexTask) runner;
+    return new ParallelIndexStats(
+        currentSequentialTask.doGetRowStats(full),
+        currentSequentialTask.doGetUnparseableEvents(full),
+        Collections.emptySet()
```

Comment:
Can you please elaborate why we are updating stats with empty intervals set here? Shouldn't we report intervals for sequential task as well?

Reply:
When `ParallelIndexSupervisorTask` is in sequential mode, it just creates an `IndexTask` and runs it.  And `IndexTask` already has logic to report intervals in its Completion report, as you saw above.  We just return row stats and unparseable events here for intermediate reports, before the task is done.

Reply:
Oh ok this is for live reports. My bad!